src/apify_helpers.js

```diff
@@ -230,22 +392,62 @@ const getActorAdditionalFields = async (z, bundle) => {
 
     let inputBody;
     let inputContentType;
-    let inputSchema;
     // Get input schema from build
-    const defaultBuild = actor.taggedBuilds && actor.taggedBuilds[defaultActorBuildTag];
-    if (defaultBuild) {
-        const buildResponse = await wrapRequestWithRetries(z.request, {
-            url: `${APIFY_API_ENDPOINTS.actors}/${actorId}/builds/${defaultBuild.buildId}`,
-        });
-        inputSchema = buildResponse.data && buildResponse.data.inputSchema;
-        if (inputSchema) {
-            inputContentType = 'application/json; charset=utf-8';
-            inputBody = JSON.stringify(getPrefilledValuesFromInputSchema(inputSchema), null, 2);
-        }
+    const inputSchema = await maybeGetInputSchemaFromActor(z, actor, defaultActorBuildTag);
```

Comment:
Is it possible to use the build value from the `bundle.inputData` if it's available, instead of just the default? So it would change the fields when they change in different builds? I see that the build has the `altersDynamicFields` on, but seems unused.

Reply:
You are right, I need to use it in case the user changes the build but doesn't change actorId.